### ISSUE TITLE
Fix Android tracker version to 1.5.x

### ISSUE THREAD

**Issue by MarlosAugusto** [snowplow-incubator/snowplow-react-native-tracker#91]
In the last [update](https://github.com/snowplow/snowplow-android-tracker/commit/5a8b9827ed75f0c16ca0b0a52587e1d99202e76e) on the native tracker for android, the "waitForEventStore" method by Emitter was removed.

**To Reproduce**
use snowplow-react-native-tracker and compile some project.

**Screenshots**
snowplow-react-native-tracker code (being used in a project):
![image](https://user-images.githubusercontent.com/42413502/92047335-409c1880-ed5b-11ea-9434-4714e37185c7.png)

build error:
![image](https://user-images.githubusercontent.com/42413502/92047644-041cec80-ed5c-11ea-85aa-e703157c610f.png)

**Smartphone:**
 - Android emulator(in this case the version or model of the smartphone is indifferent.

**Comment by colmsnowplow** [comment]
Thanks for flagging @MarlosAugusto, I think I see how that happened, working on a fix now.

**Comment by colmsnowplow** [comment]
The method in question causes the tracker to wait for the SQLite Database to initialise before proceeding, to prevent data loss. v1.6 of the android tracker uses a different DB with no need to do this, and deprecates the method.

Short term solution is to fix android tracker version to `1.5.+`, to unblock those that are having this issue. Longer term, we remove that line, use the latest version, and test that this doesn't impact anything else (separate issue for that: https://github.com/snowplow-incubator/snowplow-react-native-tracker/issues/92).

**Comment by MarlosAugusto** [comment]
OK :)

**Comment by colmsnowplow** [comment]
v0.1.1 is now published and should resolve the issue. Thanks again for reporting @MarlosAugusto

**Comment by MarlosAugusto** [comment]
amazing, thanks for answering so quickly and for nothing :D
